components/vf-button/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+### 2.0.0-alpha.1
+
+* removes deprecated code
```

Comment:
I think like the vf-badge pull request, this changelog would benefit from a mini "how to migrate" note.

Reply:
> a mini "how to migrate"

if the usage of existing primary, secondary, tertiary badges and buttons were implemented 'semantically' then I don't think there would be a need. 

So, yes - you're probably right -- but there probably needs more documentation as to when to use what where and why.

Reply:
@khawkins98 I've added the relevant migration details - I'll let you resolve this conversation and I will merge the PR once approved.

Reply:
👍